frontend/src/pages/dashboard/components/sidebar/sidebar.tsx

```diff
@@ -68,6 +69,7 @@ const Sidebar: React.FC<Properties> = ({ isCollapsed = false }: Properties) => {
       )}
     >
       <ul className={styles.list}>{renderTabs()}</ul>
+      {user?.group.key === UserGroupKey.DRIVER && <EndShiftButton />}
```

Comment:
Maybe put this condition in a variable and use it in getTabs too? 
`const isUserDriver = user?.group.key === UserGroupKey.DRIVER;`
```suggestion
      {isUserDriver && <EndShiftButton />}
```